ngraph/core/src/op/gather_nd.cpp

```diff
@@ -122,3 +66,28 @@ shared_ptr<Node> op::v5::GatherND::clone_with_new_inputs(const OutputVector& new
     check_new_args_count(this, new_args);
     return make_shared<op::v5::GatherND>(new_args.at(0), new_args.at(1), m_batch_dims);
 }
+
+// ------------------------------ V8 ------------------------------
+BWDCMP_RTTI_DEFINITION(op::v8::GatherND);
+
+op::v8::GatherND::GatherND(const Output<Node>& data, const Output<Node>& indices, const size_t batch_dims)
+    : GatherNDBase(data, indices, batch_dims) {
+    constructor_validate_and_infer_types();
+}
+
+void op::v8::GatherND::validate_and_infer_types() {
+    NGRAPH_OP_SCOPE(v8_GatherND_validate_and_infer_types);
+    validate_inputs_and_infer_shape();
+}
+
+bool op::v8::GatherND::visit_attributes(AttributeVisitor& visitor) {
+    NGRAPH_OP_SCOPE(v8_GatherND_visit_attributes);
+    visitor.on_attribute("batch_dims", m_batch_dims);
+    return true;
+}
+
+shared_ptr<Node> op::v8::GatherND::clone_with_new_inputs(const OutputVector& new_args) const {
+    NGRAPH_OP_SCOPE(v8_GatherND_clone_with_new_inputs);
+    check_new_args_count(this, new_args);
+    return make_shared<op::v8::GatherND>(new_args.at(0), new_args.at(1), m_batch_dims);
+}
```

Comment:
Looks like we can move these methods to the base class.

Reply:
Moved.